packages/malloy/src/lang/parse-malloy.ts

```diff
@@ -670,17 +670,17 @@ export abstract class MalloyTranslation {
   fatalErrors(): FatalResponse {
     return {
       final: true,
-      errors: [...this.root.logger.getLog()],
+      problems: [...this.root.logger.getLog()],
```

Comment:
filter out warnings?

Reply:
Well I did this, and it broke everything!

We're using `fatalErrors` to generate the response in the case that there _are_ fatal errors, but even then we still want to show warnings. So I renamed it to `fatalResponse`, to hopefully clarify.